### PR TITLE
Fixes Issue #434 and #680: Add Needs Contact button to Issues UI

### PR DESCRIPTION
**Pull request by miketaylr**
r? @magsout

### CODE REVIEW

**Comment by miketaylr** [comment]
Note that we're not adding to the homepage "Browse Issues" part because eventually we're going to make that only New issues.

**Comment by magsout** [comment]
ooops

File: webcompat/static/css/development/components/filter.css [hunk 1]
```diff
@@ -26,12 +26,19 @@
...
+  .wc-Filter--needsdiagnosis:hover,
+  .wc-Filter--needsdiagnosis.is-active {
+    background-color:var(--wc-state-needsdiagnosis);
+  }
+  .wc-Filter--needscontact {
+    border-color:var(--wc-state-needscontact);
+  }
+  .wc-Filter--needscontact:hover,
+  .wc-Filter--needscontact.is-active {
+    background-color:var(--wc-state-needscontact);
```
Comment (magsout) [inline webcompat/static/css/development/components/filter.css]: just a detail, but I prefer camel case

```
wc-Filter--needsContact.is-active {
  background-color:var(--wc-state-needsContact);
}
```

File: webcompat/static/css/development/components/issue-detail.css [hunk 2]
```diff
@@ -46,8 +46,12 @@
   background-color: var(--wc-state-new);
 }
 /*Needs Diagnosis*/
-.wc-IssueDetail-state--need {
-  background-color: var(--wc-state-need);
+.wc-IssueDetail-state--needsdiagnosis {
```
Comment (magsout) [inline webcompat/static/css/development/components/issue-detail.css]: Same here

**Comment by magsout** [comment]
except the small details, looks good to me

**Comment by miketaylr** [comment]
Sounds good, will tweak the case. :+1:

**Pull request merged**